price-feeder/oracle/util.go

```diff
@@ -160,7 +160,7 @@ func StandardDeviation(
 			means[base] = sdk.ZeroDec()
 		}
 
-		numPrices := int64(len(priceSlice))
+		numPrices := int64(len(priceSlice[base]))
```

Comment:
- This was affecting the calculation of `StandardDeviation`

Reply:
Please add another changelog entry with this PR under "bugs" describing that this is fixed